src/app/external-cluster-wizard/steps/external-cluster/provider/eks/component.ts

```diff
@@ -308,6 +308,7 @@ export class EKSClusterSettingsComponent
           roleArn: this.controlValue(Controls.RoleArn),
           version: version,
           vpcConfigRequest: {
+            vpcId: this.controlValue(Controls.Vpc),
```

Comment:
Will it have any effect on the API request since this isn't required in API payload?

Reply:
I've tested it, API works correctly.